libs/core/files-manager/actions/get-files.action.ts

```diff
@@ -11,23 +11,78 @@ import {
 } from "Core/files-manager/constants"
 import { getFilesRequest } from "Core/files-manager/requests/get-files.request"
 import { File } from "Core/files-manager/dto"
-import { loadDeviceData } from "Core/device"
+import { DeviceType, loadDeviceData } from "Core/device"
 import { ReduxRootState } from "Core/__deprecated__/renderer/store"
+import { getActiveDeviceTypeSelector } from "Core/device-manager/selectors/get-active-device-type.selector"
+import { FilesManagerState, SoundApp } from "Core/files-manager/reducers"
 
-export const getFiles = createAsyncThunk<File[], DeviceDirectory, { state: ReduxRootState }>(
+export const getFiles = createAsyncThunk<
+  FilesManagerState["filesMap"],
+  undefined,
+  { state: ReduxRootState }
+>(
   FilesManagerEvent.GetFiles,
-  async (payload, { rejectWithValue, dispatch }) => {
-    const result = await getFilesRequest({
-      directory: payload,
-      filter: { extensions: eligibleFormat },
-    })
-
-    if (!result.ok || !result.data) {
-      return rejectWithValue(result.error)
+  async (_, { rejectWithValue, dispatch, getState }) => {
+    const deviceType = getActiveDeviceTypeSelector(getState())
+    if (deviceType === undefined) {
+      return rejectWithValue("device Type isn't set")
+    }
+
+    const directories = getDirectoriesByDeviceType(deviceType)
+
+    const getFilesResults: FilesManagerState["filesMap"] = {}
+
+    for (const directory of directories) {
+      const result = await getFilesRequest({
+        directory,
+        filter: { extensions: eligibleFormat },
+      })
+
+      if (!result.ok || !result.data) {
+        return rejectWithValue(result.error)
+      }
+
+      const filesMapKey = getFilesMapKey(directory, deviceType)
+
+      getFilesResults[filesMapKey] = result.data as File[]
     }
 
     await dispatch(loadDeviceData())
 
-    return result.data
+    return getFilesResults
   }
 )
+
+const getFilesMapKey = (
+  directory: DeviceDirectory,
+  deviceType: DeviceType
+): SoundApp => {
+  if (
+    directory === DeviceDirectory.Music &&
+    deviceType === DeviceType.MuditaPure
+  ) {
+    return "PURE_MUSIC"
+  } else if (
+    directory === DeviceDirectory.Relaxation &&
+    deviceType === DeviceType.MuditaHarmony
+  ) {
+    return "HARMONY_RELAXATION"
+  } else if (
+    directory === DeviceDirectory.Sounds &&
+    deviceType === DeviceType.MuditaHarmony
+  ) {
+    return "HARMONY_SOUNDS"
+  } else {
+    return "UNKNOWN"
+  }
+}
+
+const getDirectoriesByDeviceType = (
+  deviceType: DeviceType
+): DeviceDirectory[] => {
+  if (deviceType === DeviceType.MuditaPure) {
+    return [DeviceDirectory.Music]
+  } else {
+    return [DeviceDirectory.Relaxation]
```

Comment:
Should it also include 'DeviceDirectory.Sounds'?

Reply:
will be add in the next task during read action